docs/changelog.rst

```diff
@@ -203,6 +204,7 @@ Changelog
 * :bug:`870` User selectable NAT traversal.
 * :feature:`921` Add ``/api/1/connection`` API endpoint returning information about all connected token networks.
 * :bug:`1011` Remove ``settled`` attribute from the NettingChannel smart contract.
+* :bug'`3022` Reject REST API channel opening with an error if there is not enough token balance for the initial deposit.
```

Comment:
Don't put this here but in the top of the file -- iow first entry after :release:`0.16.0 <2018-11-09>`